facebook/delphiFacebook/R/contingency_run.R

```diff
@@ -0,0 +1,121 @@
+#' Wrapper that runs `run_contingency_tables_one_period` over several time periods
+#'
+#' Allows pipeline to be create a series of CSVs for a range of dates.
+#'
+#' See the README.md file in the source directory for more information about how to run
+#' this function.
+#'
+#' @param params    Params object produced by read_params
+#' @param aggregations Data frame with columns `name`, `var_weight`, `metric`,
+#'   `group_by`, `compute_fn`, `post_fn`. Each row represents one aggregate
+#'   to report. `name` is the aggregate's base column name; `var_weight` is the 
+#'   column to use for its weights; `metric` is the column of `df` containing the
+#'   response value. `group_by` is a list of variables used to perform the 
+#'   aggregations over. `compute_fn` is the function that computes
+#'   the aggregate response given many rows of data. `post_fn` is applied to the
+#'   aggregate data after megacounty aggregation, and can perform any final
+#'   calculations necessary.
+#'
+#' @return none
+#' 
+#' @importFrom lubridate ymd days
+#' 
+#' @export
+run_contingency_tables <- function(params, aggregations)
+{
+  if (!is.null(params$end_date) & !is.null(params$n_periods)) {
+    ## Produce historical CSVs
+    
+    if (params$aggregate_range == "week") {
+      period_step <- days(7)
+    } else if (params$aggregate_range == "week") {
```

Comment:
shouldn't one of these be `month`? and do you need to add `months` to the `@importFrom` above?